### ISSUE TITLE
/var/www/owncloud/.htaccess missing after Debian upgrade

### ISSUE THREAD

**Issue by Dude4Linux** [owncloud/core#14601]
While upgrading from owncloud version 7.0.2 on a Debian Wheezy machine which was installed by
    apt-get install owncloud
to version 8.0 using
    apt-get install owncloud-server
the old version is removed while preserving the /data and config.php, but base .htaccess file is missing.
I can complete the upgrade by copying the file from this repository, running Update, and then correcting the file ownership and permissions.  Ideally, the Debian package should include the v8.0 .htaccess file and also make the necessary changes in ownership/permissions.

**Comment by LukasReschke** [comment]
@jnweiger

**Comment by jnweiger** [comment]
The .htaccess file is included and flagged a configfile in 8.0.2, as released today. 
That should fix the issue.
Please reopen, if needed. Thanks for the report!

**Comment by Dude4Linux** [comment]
I just re-ran the test using 8.0.2 and experienced the same result, i.e. missing .htaccess

Starting with a stable installation of 7.0.2-1 on Debian Wheezy, I run the following commands to upgrade
```
apt-get update && apt-get update && apt-get install owncloud-server
```
When I run the last step I get

```
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following extra packages will be installed:
  owncloud-3rdparty owncloud-app-activity owncloud-app-external
  owncloud-app-files owncloud-app-files-encryption owncloud-app-files-external
  owncloud-app-files-locking owncloud-app-files-pdfviewer
  owncloud-app-files-sharing owncloud-app-files-texteditor
  owncloud-app-files-trashbin owncloud-app-files-versions
  owncloud-app-files-videoviewer owncloud-app-firstrunwizard
  owncloud-app-gallery owncloud-app-provisioning-api
  owncloud-app-templateeditor owncloud-app-user-external
  owncloud-app-user-ldap owncloud-app-user-webdavauth
Suggested packages:
  php5-apcu php-apc
The following packages will be REMOVED:
  owncloud
The following NEW packages will be installed:
  owncloud-3rdparty owncloud-app-activity owncloud-app-external
  owncloud-app-files owncloud-app-files-encryption owncloud-app-files-external
  owncloud-app-files-locking owncloud-app-files-pdfviewer
  owncloud-app-files-sharing owncloud-app-files-texteditor
  owncloud-app-files-trashbin owncloud-app-files-versions
  owncloud-app-files-videoviewer owncloud-app-firstrunwizard
  owncloud-app-gallery owncloud-app-provisioning-api
  owncloud-app-templateeditor owncloud-app-user-external
  owncloud-app-user-ldap owncloud-app-user-webdavauth owncloud-server
0 upgraded, 21 newly installed, 1 to remove and 0 not upgraded.
Need to get 23.5 MB of archives.
```
I think the problem may be that .htaccess is not marked as a config file in the owncloud 7.x.x package and hence it is removed when the owncloud package is uninstalled.  The owncloud-server packages need to be smart enough to replace the file whenever is isn't present.
